README.md

```diff
@@ -74,137 +95,63 @@ cd deephaven-deployment
 
 > :warning: **Commands in the following sections for interacting with a deployment must be run from the deployment directory.**
 
-### Launch: Python
-
-Run the following commands to launch Deephaven for Python server applications. 
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python/base/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
-
-### Launch: Python with NLTK
-
-Run the following commands to launch Deephaven for Python server applications with the [NLTK](https://nltk.org/) module pre-installed.
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python/NLTK/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
-
-### Launch: Python with PyTorch
-
-Run the following commands to launch Deephaven for Python server applications with the [PyTorch](https://pytorch.org/) module pre-installed.
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python/PyTorch/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
-
-### Launch: Python with SciKit-Learn
-
-Run the following commands to launch Deephaven for Python server applications with the [SciKit-Learn](https://scikit-learn.org/stable/) module pre-installed.
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python/SciKit-Learn/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
-
-### Launch: Python with TensorFlow
-
-Run the following commands to launch Deephaven for Python server applications with the [TensorFlow](https://www.tensorflow.org/) module pre-installed.
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python/TensorFlow/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
-
-### Launch: Python with example data
+### Launch options
 
-Run the following commands to launch Deephaven for Python server applications, with example data.
+Deephaven offers a variety of pre-made Docker Compose YAML files for different deployments. The following commands will launch Deephaven. Replace `<URL>` with that of the deployment from the table below that best suits your needs.
```

Comment:
I think "YAML" should either say what the acronym stands for in parentheses, or just call it a .yml file.